package.json

```diff
@@ -18,6 +18,7 @@
 		"tailwindcss": "^3.3.2"
 	},
 	"devDependencies": {
+		"@smui/data-table": "^7.0.0-beta.8",
```

Comment:
Why add SMUI and not use what we already have with Tailwind: https://tailwindui.com/components/application-ui/lists/tables?

Reply:
SMUI is Svelte Material UI, and TailwindCSS is not TailwindUI, we haven't bought it
In Svelte there is one interesting thingy about its libraries -- they are all are divided into tiny pieces